src/main/java/svc/data/municipal/MunicipalityDAO.java

```diff
@@ -53,10 +53,18 @@ public Municipality getByMunicipalityId(Long municipalityId){
 		}
 	}
 	
-	public List<Municipality> getAllMunicipalities() {
+	public List<Municipality> getAllMunicipalities(Boolean getSupportedOnly){
```

Comment:
getSupportedOnly doesn't make sense here. You're saying get supported only true and it means get only the supported. That's fine. But you're saying get supported only false and that really means get all, not get all unsupported. I'd keep the variable name just `supported` just like you have in the controller